### PR TITLE
List WrkzCoin (WRKZ)

### PR DESCRIPTION
**Pull request by wrkzdev**
This is a re-submission https://github.com/bisq-network/bisq/pull/1834

- Official project URL: https://www.wrkz.work/
- Official block explorer URL: https://myexplorer.wrkz.work/

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/WrkzCoin.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class WrkzCoin extends Coin {
+
+    public WrkzCoin() {
+        super("WrkzCoin", "WRKZ", new RegexAddressValidator("^Wrkz[1-9A-Za-z^OIl]{94}"));
+    }
+}
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/WrkzCoin.java]: Add newline at the end of file

File: assets/src/test/java/bisq/asset/coins/WrkzCoinTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    @Test
+    public void testInvalidAddresses() {
+        assertInvalidAddress("WrkzQokcStLUSALE5Ra17v2n6ad65h8wL5vqABKkoWy7Xicz9znqPSgS2MRVkuYtRAaJiMFuyDCFF1oJgT7PHb8i9yM");
+        assertInvalidAddress("WrkskixT63cYzLFmDoA5WN7RbihYBwbzJJmjR9zgjD3ZUotbFGBgv1RaUAu1fWWT4QeEEktQfZK9AFPh19t2U8uG49EH3WSVEn");
+        assertInvalidAddress("");
+        assertInvalidAddress("WrkzUAxg9TSdkh6tfh5pk84XgKeyNe8T4TvaSgk87kk6iCUEitkk2sk6wVtKJXk5BM3kwh2ftnkaVfBWfBPr8igZ2xkn#RoUxF");
+        assertInvalidAddress("WrkzXTU4REbRijuLPpds2k4BhcBGgXFpeEaXKs49D7$PFuqBYpQw2tQbAApoQLAp2iWVsoxiPmcERXhHrhtCLnzL4ezB8kAbxH");
+        assertInvalidAddress("cccd2bd37455350e7586cf9315c7f3acd3de56321aa356ff3391bd21f0bbf502");
+    }
+}
```
Comment (blabno) [inline assets/src/test/java/bisq/asset/coins/WrkzCoinTest.java]: Add newline at the end of file

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Review by blabno — APPROVED** [review]
ACK

**Comment by ripcurlx** [comment]
@wrkzdev Please fix the conflict and I'll merge the PR

**Pull request merged**